New shrine icon

endim8 opened the pull request:
day/night
<img width="468" alt="Screenshot 2022-12-07 at 11 24 05" src="https://user-images.githubusercontent.com/26939824/206167713-8ad0f100-0ddf-4f0f-8bf7-b7fecda7077a.png">


fixes https://github.com/organicmaps/organicmaps/issues/3984

biodranik commented:
Maybe something like this?

https://as1.ftcdn.net/v2/jpg/03/09/62/06/1000_F_309620638_yS37tDDYLakIexCpc3HdRWVMp904bTgQ.jpg

endim8 commented:
> Not sure if a shrine is more recognizable.

They are quite small usually, not a building:
<img width="468" src="https://user-images.githubusercontent.com/26939824/206194768-6db15e3c-f73a-405f-b997-44a60ab00bf1.png">

(although we could add your suggested icon for `historic=wayside_shrine` + `building=shrine`)

endim8 commented:
> (from issue) Carto icon looks more like a monument )

this is true, i'll make some variants

endim8 commented:
how about this?:

<img width='64' src="https://user-images.githubusercontent.com/26939824/206203204-12046bf9-d868-4e82-8fe6-c42b59a8724d.svg">

<img width="234" alt="Screenshot 2022-12-07 at 14 27 18" src="https://user-images.githubusercontent.com/26939824/206204984-6f8189bf-c606-4d89-80b4-e4a383cdfb30.png">

endim8 commented:
now using new icon

endim8 commented:
@vng i've fixed the icon for `amenity=post_office`+`shop=copyshop`, but the element type is still copyshop after map regeneration. Where do you set the priority for element types? (issue: https://github.com/organicmaps/organicmaps/issues/4025)

dbf256 commented:
The new bar icon is currently used for nightclubs, maybe better not to mix them?
![Screenshot_2022-12-07-22-07-20-299_app organicmaps](https://user-images.githubusercontent.com/2550295/206295912-a1124eb8-c1b3-44aa-b5d3-963489f0f766.png)

vng commented:
Is it possible to make pretty but recognizable and different icons for:
bar, pub, nightclub, biergarten
?

dvdmrtnz commented:
> Is it possible to make pretty but recognizable and different icons for: bar, pub, nightclub, biergarten ?

The current state of these icons in Organic Maps is:

| Tag | Icon |
|-|-|
| amenity=nightclub | <img src="https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/clear/style-clear/symbols/bar-m.svg" width="35"/>
| amenity=bar | <img src="https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/clear/style-clear/symbols/beer-m.svg" width="35"/>
| amenity=pub | <img src="https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/clear/style-clear/symbols/beer-m.svg" width="35"/>
| amenity=biergarten | <img src="https://raw.githubusercontent.com/organicmaps/organicmaps/master/data/styles/clear/style-clear/symbols/beer-m.svg" width="35"/>

In OSM carto they use these icons:

| Tag | Icon |
|-|-|
| amenity=nightclub | <img src="https://wiki.openstreetmap.org/w/images/e/ee/Nightclub-16.svg" width="35"/>
| amenity=bar | <img src="https://wiki.openstreetmap.org/w/images/9/94/Bar-16.svg" width="35"/>
| amenity=pub | <img src="https://wiki.openstreetmap.org/w/images/5/5d/Pub-16.svg" width="35"/>
| amenity=biergarten | <img src="https://wiki.openstreetmap.org/w/images/e/e1/Biergarten-16.svg" width="35"/>

endim8 commented:
i think copying carto would be best 👍 i'll work on the icons in a bit

endim8 commented:
> how about this?:
> 
> <img alt="" width="64" src="https://user-images.githubusercontent.com/26939824/206203204-12046bf9-d868-4e82-8fe6-c42b59a8724d.svg">
> 
> <img alt="" width="64" src="https://user-images.githubusercontent.com/26939824/206206582-b867b3f8-ca4e-47a7-b3ba-9347fcdb8e42.svg">
> 
> <img alt="Screenshot 2022-12-07 at 14 27 18" width="234" src="https://user-images.githubusercontent.com/26939824/206204984-6f8189bf-c606-4d89-80b4-e4a383cdfb30.png">

feedback on this would be good, anything need changing?

endim8 commented:
new:
Tag | Icon
-- | --
amenity=nightclub |  ![nightclub-m](https://user-images.githubusercontent.com/26939824/206478988-8251c281-3395-4164-bd38-6e151a7a41ff.svg)
amenity=bar |  ![bar-m](https://user-images.githubusercontent.com/26939824/206476088-f9b9ff7b-faab-4090-8364-60a20e660ade.svg)
amenity=pub, biergarten |  ![beer-m](https://user-images.githubusercontent.com/26939824/206475961-4eede965-5e42-4bd1-a45b-55e6ab3d398a.svg)

@biodranik PTAL :)

endim8 commented:
Changed priority for historic=ship to render over tourism=attraction:
![Screenshot 2022-12-08 at 15 08 34](https://user-images.githubusercontent.com/26939824/206482814-50784e2f-2a44-4fa0-81ef-b1c065b68dea.png)

(colour will be fixed in https://github.com/organicmaps/organicmaps/pull/4046)

biodranik commented:
> > how about this?:
> > <img alt="" width="64" src="https://user-images.githubusercontent.com/26939824/206203204-12046bf9-d868-4e82-8fe6-c42b59a8724d.svg">
> > <img alt="" width="64" src="https://user-images.githubusercontent.com/26939824/206206582-b867b3f8-ca4e-47a7-b3ba-9347fcdb8e42.svg">
> > <img alt="Screenshot 2022-12-07 at 14 27 18" width="234" src="https://user-images.githubusercontent.com/26939824/206204984-6f8189bf-c606-4d89-80b4-e4a383cdfb30.png">
> 
> feedback on this would be good, anything need changing?

Need to improve it, now it looks like a house.

endim8 commented:
> Need to improve it, now it looks like a house.

@biodranik How about this?

<img width="175" alt="Screenshot 2022-12-08 at 22 39 00" src="https://user-images.githubusercontent.com/26939824/206582123-18697a07-bf26-4e93-95d7-87251ab20dc2.png">


<img width="128" src="https://user-images.githubusercontent.com/26939824/206581497-57f19ace-fb79-4917-aadf-88f6a8857958.svg"/>

it matches the tomb icon:
<img width="128" src="https://user-images.githubusercontent.com/26939824/206579102-2142173e-5c0c-4c21-b675-98577379f405.svg"/>

biodranik commented:
... and now it's a tomb with a monument :)

endim8 commented:
> ... and now it's a tomb with a monument :)

aaargh 😅 maybe i should move it to a new PR? it needs more consideration

biodranik commented:
Yes, we can improve it separately )

biodranik reviewed (approved):
It would be great to update icons wiki page.

Re: wayside shrine: Maybe something like this?

![image](https://user-images.githubusercontent.com/170263/206683231-4316bad4-b257-4fa1-a65d-d53ce5823bc2.png)

dvdmrtnz commented:
> It would be great to update icons wiki page.

Done

endim8 commented:
> > It would be great to update icons wiki page.
> 
> Done

thank you 👍 i don't seem to have access to edit the wiki

rtsisyk commented:
Folks, could you please point me to the source of your new icons? We need to check that copyrights are ok.

endim8 commented:
> Folks, could you please point me to the source of your new icons? We need to check that copyrights are ok.

none of the icons used in this PR are new, I just changed the drules. In the case of music shop & nightclub, i reused the `shop=musical_instrument` icon with different colours (and replaced musical_instrument with a previously unused icon). The currently unused shrine icons are made by me, so CC0 :)
